### ISSUE TITLE
<module:XMLRPCRetryable>': uninitialized constant Net::ReadTimeout (NameError)

### ISSUE THREAD

**Issue by egy-shell** [Hood3dRob1n/creep3r#1]
./creep3r
/var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress/xml_rpc_retryable.rb:7:in `<module:XMLRPCRetryable>': uninitialized constant Net::ReadTimeout (NameError)
        from /var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress/xml_rpc_retryable.rb:2:in`module:Rubypress'
        from /var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress/xml_rpc_retryable.rb:1:in `<top (required)>'
        from /var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress/client.rb:11:in`require_relative'
        from /var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress/client.rb:11:in `<top (required)>'
        from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in`require'
        from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
        from /var/lib/gems/1.9.1/gems/rubypress-1.0.7/lib/rubypress.rb:4:in`<top (required)>'
        from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:60:in `require'
        from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:60:in`rescue in require'
        from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:35:in `require'
        from ./creep3r:44:in`<main>'

how 2 fix this

**Comment by shadowzoom** [comment]
I have problems with this ruby to....I have Kali Linux. Hood3dRob1n please check and tell us what we need do in Kali Linux. Try install creep3r on clean just installed Kali, i don't know what to do.Im tryed install various rubyes, buts everytime i get fail. Help please bro.

**Comment by Hood3dRob1n** [comment]
The rubypress gem has some calls which are not properly checked first which results in the error thrown on certain versions of Ruby. As I had not fully committed this piece into the framework I have removed all references to it. I just tested this out on fresh Kali install and appears to have resolved the error. I will be working on my own implementation to re-include this feature at a later date. Thanks for the heads up and sorry about the issues. Let me know if you run into anything else...
